h/templates/app.html.jinja2

```diff
@@ -1,127 +1,93 @@
+    {% for template in angular_templates -%}
+      <script type="text/ng-template" id="{{template.name}}">
+        {{ template.content | safe }}
+      </script>
+    {% endfor -%}
 
-{% block templates %}
-  <script type="text/ng-template" id="annotation.html">
-    {{ include_raw("h:templates/client/annotation.html") }}
-  </script>
-  <script type="text/ng-template" id="excerpt.html">
-    {{ include_raw("h:templates/client/excerpt.html") }}
-  </script>
-  <script type="text/ng-template" id="group_list.html">
-    {{ include_raw("h:templates/client/group_list.html") }}
-  </script>
-  <script type="text/ng-template" id="markdown.html">
-    {{ include_raw("h:templates/client/markdown.html") }}
-  </script>
-  <script type="text/ng-template" id="share_dialog.html">
-    {{ include_raw("h:templates/client/share_dialog.html") }}
-  </script>
-  <script type="text/ng-template" id="viewer.html">
-    {{ include_raw("h:templates/client/viewer.html") }}
-  </script>
-  <script type="text/ng-template" id="thread.html">
-    {{ include_raw("h:templates/client/thread.html") }}
-  </script>
-  <script type="text/ng-template" id="dropdown_menu_btn.html">
-    {{ include_raw("h:templates/client/dropdown_menu_btn.html") }}
-  </script>
-  <script type="text/ng-template" id="publish_annotation_btn.html">
-    {{ include_raw("h:templates/client/publish_annotation_btn.html") }}
-  </script>
-  <script type="text/ng-template" id="search_status_bar.html">
-    {{ include_raw("h:templates/client/search_status_bar.html") }}
-  </script>
-  <script type="text/ng-template" id="sidebar_tutorial.html">
-    {{ include_raw("h:templates/client/sidebar_tutorial.html") }}
-  </script>
-  <script type="text/ng-template" id="signin_control.html">
-    {{ include_raw("h:templates/client/signin_control.html") }}
-  </script>
-  <script type="text/ng-template" id="sort_dropdown.html">
-    {{ include_raw("h:templates/client/sort_dropdown.html") }}
-  </script>
-  <script type="text/ng-template" id="top_bar.html">
-    {{ include_raw("h:templates/client/top_bar.html") }}
-  </script>
-{% endblock %}
+    <!-- App Configuration !-->
+    <script id="hypothesis-settings" type="application/json">
+      {
+        "apiUrl": "{{ api_url }}",
+        "websocketUrl": "{{ websocket_url }}"
+      }
+    </script>
 
-{# Override ga_pageview block to prevent a 'pageview' event from being sent to
-   Google Analytics #}
-{% block ga_pageview %}{% endblock %}
+    <!-- Scripts !-->
+    {% for url in app_js_urls %}
+    <script src="{{ url }}"></script>
+    {% endfor %}
 
-{% block scripts %}
-  <script id="hypothesis-settings" type="application/json">
-    {
-      "apiUrl": "{{ request.route_url('api') }}",
-      "websocketUrl": "{{ request.route_url('ws') | websocketize }}"
-    }
-  </script>
-  {% assets "app_js" %}
-    <script src="{{ ASSET_URL }}"></script>
-  {% endassets %}
-{% endblock %}
+    <!-- Analytics !-->
+    {% if ga_tracking_id %}
```

Comment:
I'm not passing in 'ga_tracking_id' here. I'm just checking whether we still need this.